javascript-modal/main.js

```diff
@@ -0,0 +1,12 @@
+var modal = document.querySelector('.modal');
+var trigger = document.querySelector('.trigger');
+var noButton = document.querySelector('.no-button');
+
+trigger.addEventListener('click', function () {
+  modal.classList.toggle('show-modal');
+
+});
```

Comment:
Instead of using two anonymous functions with the same code, make one named function that you can use as the event handler for both of these buttons